bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/channel/GroupAddressConfiguration.java

```diff
@@ -12,41 +12,106 @@
  */
 package org.openhab.binding.knx.internal.channel;
 
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import tuwien.auto.calimero.GroupAddress;
+import tuwien.auto.calimero.KNXFormatException;
 
 /**
- * Data structure representing a single group address configuration within a channel configuration parameter.
+ * Data structure representing the content of a channel's group address configuration.
  *
  * @author Simon Kaufmann - initial contribution and API.
  *
  */
 @NonNullByDefault
 public class GroupAddressConfiguration {
+    public static final Logger LOGGER = LoggerFactory.getLogger(GroupAddressConfiguration.class);
```

Comment:
Loggers should be non-static. For reference: https://www.openhab.org/docs/developer/guidelines.html#f-logging

Reply:
Not possible because the logger is used in static method `parse`.